src/Storage/Storage.Management/help/Close-AzStorageFileHandle.md

```diff
@@ -74,7 +74,7 @@ This command closes all file handles on a file.
 
 ### Example 2: Close all file handles which is opened 1 day ago on a file directory
 ```
-PS C:\> Get-AzStorageFileHandle -ShareName "mysharename" -Path 'dir1/dir2' -Recursive | ? {$_.OpenTime.DateTime.AddDays(1) -lt (Get-Date)} | Close-AzStorageFileHandle -ShareName "mysharename"
+PS C:\> Get-AzStorageFileHandle -ShareName "mysharename" -Path 'dir1/dir2' -Recursive | Where-Object {$_.OpenTime.DateTime.AddDays(1) -lt (Get-Date)} | Close-AzStorageFileHandle -ShareName "mysharename"
```

Comment:
The before script also work with "?", please indicate why need change to "Where-Object"

Reply:
? is the alias for Where-Object